### PR TITLE
Condensed SCXML sections to a single page

### PR DESCRIPTION
**Pull request by mattpocock**
https://www.loom.com/share/c235a583ec524238baeb4cac787cda3c

### CODE REVIEW

**Comment by laurakalbag** [comment]
> https://www.loom.com/share/c235a583ec524238baeb4cac787cda3c

Very rustly microphone in this one!

**Review by laurakalbag — COMMENTED** [review]
The restructure is great. I just added some very basic copy changes to the original text from the SCXML sections.

File: docs/guides/scxml.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
+# SCXML
+
+XState is compliant with the [SCXML (State Chart XML : State Machine Notation for Control Abstraction) specification](https://www.w3.org/TR/scxml/). This page contains details on where our API relates to the SCXML specification.
```
Comment (davidkpiano) [inline docs/guides/scxml.md:3]: New language here to suggest that XState is not strictly an SCXML interpreter (it already has more flexibility and slightly different semantics in places) but a statechart interpreter that _can_ interpret SCXML documents (cc. @Andarist)

```suggestion
XState is compatible with the [SCXML (State Chart XML: State Machine Notation for Control Abstraction) specification](https://www.w3.org/TR/scxml/). This page contains details on where our API relates to the SCXML specification.
```

**Review by davidkpiano — APPROVED** [review]
Tiny copy change; LGTM

**Pull request merged**